index.html

```diff
@@ -0,0 +1,226 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="styles.css" />
+    <title>TURN sample get relay configuration</title>
+    <style>
+   *{
+      font-family: Arial, Helvetica, sans-serif;
+    }
+    html,
+    body {
+      margin: 0;
+      border: 0;
+      padding: 0;
+      background-color: #fff;
+    }
+    main {
+      margin: auto;
+      width: 50%;
+      padding: 20px;
+      text-align: center;
+    }
+    h1 {
+      font-size: 3.5em;
+    }
+    </style>
+  </head>
+
+  <body>
+    <main>
+      <h1>TURN sample: Get a relay configuration and use it to stream</h1>
+      <p>Loading content from the API: <b id="message"></b></p>
+      <h1>Video Chat</h1>
+      <video id="local-video" height="150" autoplay></video>
+      <video id="remote-video" height="480" autoplay></video>
+    
+      <div>
+        <button id="get-video">Get Video</button>
+        <button id="call">Call</button>
+      </div>
+    </main>
+    <script src="https://cdn.socket.io/3.1.1/socket.io.min.js" integrity="sha384-gDaozqUvc4HTgo8iZjwth73C6dDDeOJsAgpxBcMpZYztUfjHXpzrpdrHRdVp8ySO" crossorigin="anonymous"></script>
+    <script type = "module">
+
+    const VideoChat = {
+        // Initialize our connection to the WebSocket.
+        socket: io(),
+      
+        // Call to getUserMedia (provided by adapter.js for cross browser compatibility)
+        // asking for access to both the video and audio streams. If the request is
+        // accepted callback to the onMediaStream function, otherwise callback to the
+        // noMediaStream function.
+        requestMediaStream: function(event){
+            navigator.mediaDevices.getUserMedia({video: true, audio: true})
+            .then(VideoChat.onMediaStream)
+            .catch(function(err) {
+                console.log(err.name + ": " + err.message);
+            });
+        },
+      
+        // The onMediaStream function receives the media stream as an argument.
+        onMediaStream: function(stream){
+          console.log('\nOnMedia\n');
+          
+          // Get the video element.
+          VideoChat.localVideo = document.getElementById('local-video');
```

Comment:
Is there a need to make it a module like this?  Could it instead be an ES6 class where these are defined as arrow functions and instead of saying "VideoChat.x" you say "this.x"?